lib/src/controls/better_player_controls_configuration.dart

```diff
@@ -76,7 +77,10 @@ class BetterPlayerControlsConfiguration {
   final Duration controlsHideTime;
 
   ///Custom controls, it will override Material/Cupertino controls
-  final Widget customControls;
+  final Widget Function(BetterPlayerController controller) customControls;
+
+  ///Flag to use a forced platform
+  final PlayerPlatform playerPlatform;
```

Comment:
Basically I think that we shouldn't configure player theme based on platform rather than platform theme.

Reply:
Also please set default theme based on platform i.e.:
PlayerTheme.material for Android
PlayerTheme.cupertino for iOS